spk/sickchill/Makefile

```diff
@@ -1,5 +1,5 @@
 SPK_NAME = sickchill
-SPK_VERS = v2020.11.24-1
+SPK_VERS = $(shell date +%Y%m%d)
```

Comment:
All that was needed is to remove the `v` oh well. :shrug:

All Synology cares about is the numbers are increasing with no letters (they mess up the version check).

![Screenshot from 2021-03-14 12-58-01](https://user-images.githubusercontent.com/5497998/111054973-e813a980-8468-11eb-8adf-82b4e699ccc6.png)

![Screenshot from 2021-03-14 13-01-21](https://user-images.githubusercontent.com/5497998/111055000-34f78000-8469-11eb-9963-37f90c0b75ac.png)

Reply:
Since the package version is always the build date, this works. Whenever the package is built it contains a snapshot of master at that date.

Reply:
True that will work. Since the package is set to a specific commit I would have keep the specific version IMHO.